modules/database/variables.tf

```
@@ -139,3 +139,14 @@ variable "tags" {
   type        = map(string)
   default     = {}
 }
+
+variable "min_capacity" {
+  description = "Minimal capacity that database will always have allocated, if not paused. Only settable for Serverless databases."
```

Update description to match usual phrasing:

```suggestion
  description = "Minimal capacity that this SQL database will always have allocated if not paused. Value must be greater than 0 is using a SKU in the serverless compute tier."
```